runtime/compiler/control/J9Options.cpp

```diff
@@ -2180,8 +2179,17 @@ bool J9::Options::feLatePostProcess(void * base, TR::OptionSet * optionSet)
       #endif
       }
 
-   if ((*vmHooks)->J9HookDisable(vmHooks, J9HOOK_VM_EXCEPTION_CATCH) ||
-       (*vmHooks)->J9HookDisable(vmHooks, J9HOOK_VM_EXCEPTION_THROW))
+   bool disableThrowToGoto = false;
+   if ((*vmHooks)->J9HookDisable(vmHooks, J9HOOK_VM_EXCEPTION_CATCH))
+      {
```

Comment:
FYI, once #3463 is merged, we also need set `J9HookExceptionCatch` here. i.e. adding one line `self()->enableHook(J9HookExceptionCatch);`.

Reply:
@0dvictor Don't forget we need to maintain the stack maps if either exception catch or throw are hooked.

Reply:
FYI #3463 has merged. We need set `self()->setReportByteCodeInfoAtCatchBlock();` here and for throw hook.

Reply:
Thanks, this PR has been updated to include the new call.